docs/lazy-adr/adr-002-ipds-da-sampling.md

```diff
@@ -185,15 +185,6 @@ func GetLeafData(
     leafIndex uint32,
     totalLeafs uint32, // this corresponds to the extended square width
 ) ([]byte, error)
-
-// PutLeaves takes the namespaced leaves, a row of the from the extended data square,
-// and calls nodes.DataSquareRowOrColumnRawInputParser of the ipld plugin.
-// The resulting ipld nodes are passed to a Batch calling AddMany:
-// https://github.com/ipfs/go-ipld-format/blob/d2e09424ddee0d7e696d01143318d32d0fb1ae63/batch.go#L29
-// Note, that this method could also return the row and column roots.
-// Tha caller is responsible for making sure that the leaves are sorted by namespace ID.
-// The data will be pinned by default.
-func PutLeaves(ctx contex.Context, namespacedLeaves [][]byte) error
```

Comment:
The idea behind this was that we have this lower level method that has ipfs, contexts etc as a dependency and PutBlock would use that. But maybe that is still not optimal as it would just be an indirection and we'd still have a dependency to networking code in `types`. ref #196 
It's good to delete this for now. When we move the PutBlock method out, we can update the ADR accrodingly.

Reply:
Noted in #196. I think we should definitely add it back when we move `PutBlock`.